### ISSUE TITLE
Update to pyarrow 2.0

### ISSUE THREAD

**Issue by Thomas-Hirsch** [awslabs/aws-data-wrangler#436]
As title says, pyarrow 2.0 has been released: https://arrow.apache.org/blog/2020/10/22/2.0.0-release/. Would be good to upgrade, as currently it's pinned to ~=1.0.0

**Comment by igorborgest** [comment]
Thank you @Thomas-Hirsch!

This upgrade already was applied here in our git repository.

Do you mind to install it directly from our repository and check if it is everything fine for your use case?
The idea is to publish it in the version `1.10.0` this weekend.

```sh
pip install git+https://github.com/awslabs/aws-data-wrangler.git --no-use-pep517
```

**Comment by igorborgest** [comment]
Released on [1.10.0](https://github.com/awslabs/aws-data-wrangler/releases/tag/1.10.0)
